roles/openshift_setup/README.md

```diff
@@ -15,3 +15,14 @@ should be configured for in an OCP/CRC cluster.
 * `cifmw_openshift_setup_ca_bundle_path`: (String) Path to the CA bundle.
 Defaults to `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem`. Only has an
 effect if `cifmw_openshift_setup_ca_registry_to_add` is set.
+* `cifmw_openshift_setup_digest_mirrors`: (List) List of alternative mirrored repository locations. Defaults to `[]`.
+    * Example:
+        ```yaml
+        cifmw_openshift_setup_digest_mirrors:
+        - mirrors:
+            - mirror.quay.io
+            source: quay.io
```

Comment:
is the spacing intentional here it looks a bit off?

```
        - mirrors:
            - mirror.quay.io
              source: quay.io
```

Reply:
Fixed, I also change the order between `source` and `mirror`, to avoid confusion